src/cr/cube/cubepart.py

```diff
@@ -526,9 +526,12 @@ def row_std_dev(self):
     @lazyproperty
     def row_std_err(self):
         """2D np.float64 ndarray of standard errors for row percentages."""
-        # --- We need to add `np.newaxis` to cast the rows margin vector to an actual
-        # --- column, in NumPy terms, to be able to devide correctly.
-        return np.sqrt(self._row_variance / self.rows_margin[:, np.newaxis])
+        rows_margin = (
+            self.rows_margin  # --- no need to cast (MR dim involved)
+            if len(self.rows_margin.shape) > 1
+            else self.rows_margin[:, np.newaxis]  # --- cast to actual vector column
+        )
```

Comment:
@slobodan-ilic can you explain the rationale of this condition? I'm not aware of the problem